database/postgres/postgres_game.py

```diff
@@ -0,0 +1,102 @@
+from typing import Dict
+from datetime import datetime
+from database.models import Games
+from database.postgres.postgres_country import save_country
+from database.postgres.postgres_team import save_team
+from database.postgres.postgres_league import save_league
+
+def save_games(json_data: Dict, sport_id, session) -> None:
+    games = json_data.get("response", [])
+    if not games:
+        print('insufficient data')
+        return
+
+    for game in games:
+        if "fixture" in game or "game" in game:
```

Comment:
DAL (DataAccessLayer) should not be handling such logic (parsing something).

The general, most wanted flow is:
`I get a model -> I verify a model -> I CREATE/UPDATE/DELETE the model entry from the database.`